### ISSUE TITLE
Database driver issue..

### ISSUE THREAD

**Issue by TheFatherMind** [loewexy/pdnsmanager#40]
I installed the stable and got those header errors.  so I installed the master and still got it so for fun I did a fresh git pull and could not could not complete the install...

![2017-07-09_20-26-49](https://user-images.githubusercontent.com/11869399/28002015-0ae83f8a-64e6-11e7-8ec4-bcb7d9a60121.png)

**Comment by loewexy** [comment]
This seems to be a problem with your system configuration. PHP is not able to find the PDO Driver for MySQL do you have the php mysql package installed. Also make shure the PDO MySQL driver is loaded by your PHP configuration.

**Comment by TheFatherMind** [comment]
I seem to have PDO support installed and enabled.  And to be clear.. this only happened when I pulled from git directly.  I will try it again and check my error logs as well.

![pdo support](https://user-images.githubusercontent.com/11869399/28032525-a53b4bc6-655f-11e7-98bb-e1f2cd7187a4.png)

**Comment by TheFatherMind** [comment]
Okay IDK what that was initially about.  I tried replicating it again in a fresh folder and nothing happens.  I press the install button and nothing happens.. does not even log a an error in my logs..  I tried it a few times...  Even tried it from an incognito window and iexplore.  I will start a new issue instance for this.

![install screenshot](https://user-images.githubusercontent.com/11869399/28034859-54e663c4-6567-11e7-9a4e-02a8725ece6b.png)
